packages/kit-headless/src/components/popover/popover.tsx

```diff
@@ -4,14 +4,7 @@ import { PopoverImpl } from './popover-impl';
 import { PopoverImplProps } from './popover-impl';
 import { FloatingProps } from './floating';
 
-type PopoverProps = PopoverImplProps &
-  (
-    | {
-        floating?: false | undefined;
-        anchorRef: never;
-      }
-    | ({ floating?: true } & FloatingProps)
-  );
+type PopoverProps = PopoverImplProps & ({ floating?: true } & FloatingProps);
```

Comment:
These types will always include FloatingProps, which is not what we want.

What's wrong with the previous types?

Reply:
It wasn't working anymore :/